modules/ocf_backups/files/rsnapshot.conf

```diff
@@ -61,6 +61,8 @@ backup	ocfbackups@ldap:/var/backups/ldap/	servers/ldap/
 backup	ocfbackups@puppet:/etc/puppetlabs/	servers/puppet/
 backup	ocfbackups@puppet:/opt/puppetlabs/	servers/puppet/
 
+backup	ocfbackups@puppetdb:/etc/puppetlabs/puppetdb/ssl/	servers/puppetdb/
```

Comment:
This is only backed up because this cert was manually generated to include the DNS SANs for `puppetdb`, `thunder`, and `puppetdb.ocf.berkeley.edu` alongside the default `thunder.ocf.berkeley.edu` (FQDN).